modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new `tablecellborderstyle` toolbar button and menu item to change table cell border style #TINY-7478
 - Added a new `tablecaption` toolbar button and menu item to toggle captions on tables #TINY-7479
 - Added a new `mceTableToggleCaption` command that toggles captions on a selected table #TINY-7479
+- Added a new `tablecellbordercolor` toolbar button and menu item to select table cell border colors #TINY-7480
+- Added a new `table_cell_border_color_map` option to define colors used by `tablecellbordercolor` buttons #TINY-7480
+- Added a new `tablecellbackgroundcolor` toolbar button and menu item to select table cell background colors #TINY-7480
+- Added a new `table_cell_background_color_map` option to define colors used by `tablecellbackgroundcolor` buttons #TINY-7480
+- Added a new `initData` to `fancymenuitem` to determine colors used by `tablecellbackgroundcolor` and `tablecellbordercolor` #TINY-7480
```

Comment:
Seeing as we don't document this menu type at all, I'm not sure we should have a changelog entry here. And if we did have a changelog entry here, it shouldn't have the `to determine colors used by ...` bit (just replace it with "to pass extra data needed for rendering" or something).

Also, it might be worth instead putting this in the bridge changelog?